### ISSUE TITLE
Remove the docs folder from the tracked files

### ISSUE THREAD

**Issue by MangelMaxime** [fable-compiler/fable-doc#10]
At first, I thought having the `docs` folder in the repo would make updating the website easier but the problem is the PR then get really messy to review.

I will remove the `docs` folder.

ATM publishing the website will need to run 2 commands because Nacara doesn't understand yet `build` mode. 

But when `build` mode will be supported publishing with be a single command. And we should even be able to do it from a CI to automate the process.

**Comment by alfonsogarciacaro** [comment]
Yes, you can ignore the folder with built documents in the dev/master branch, they only need to be committed in the gh-pages branch. fable-compiler.github.io does that (it uses the `gh-pages` npm package for publishing), and uses a simple live static server to check how things look before publishing: https://github.com/fable-compiler/fable-compiler.github.io/blob/a048a88765e776dba10918083b9bc4a7223b3095/package.json#L4-L13

**Comment by alfonsogarciacaro** [comment]
I'd be in favor of removing the `docs` folder, the diffing is not very nice for the generated files.

**Comment by MangelMaxime** [comment]
It's planned :) I have again time to work on OSS project so hopefully, I will be able to add it.
